data/deep missions.txt

```diff
@@ -0,0 +1,1366 @@
+	on accept
+		event "clear arculus"
+	
+	npc "scan outfits"
+		government Remnant
+		personality staying uninterested
+		fleet
+			names remnant
+			variant
+				"Albatross"
+				"Starling"
+		dialog `You have finished scanning the Remnant ships in the system. You may now return to <destination> with the scanner logs.`
+	
+	on fail
+		event "reset arculus"
+	
+	on complete
+		event "reset arculus"
+		event "deep: scan log research" 21
+		dialog
+			`The team is visibly excited when you return with the scanner logs. "Give us a few weeks to look these over," Ivan says, "then we will contact you about what we want you to do next."`
+
+
+
+mission "Deep: Remnant 1B"
+	landing
+	name `Scanning Remnant`
+	description `Scan the outfits of the Remnant ships in Arculus, then return to <destination> so that the scientists may analyze your scanner logs.`
+	source Valhalla
+	waypoint Arculus
+	to offer
+		has "Deep: Remnant: Keystone Research: done"
+		has "knew of stones before scientists"
+	on offer
+		conversation
+			`You contact Ivan after you land and he sends you directions to his research lab, which is located on the outskirts of the spaceport city. When you arrive you are greeted by Ivan, Maria, and a number of other scientists on the team. Ivan begins to inform you on what their research has revealed.`
+			`	"We've analyzed the Quantum Keystones, and they do allow entry into the wormhole in Terminus just as you said. It would appear that the stones stabilize the wormhole as the ship passes through it, or maybe it is stabilizing the ship in some way and not the wormhole. We'll need much more time to fully understand the true mechanism that is at play here. Whatever the case is, one stone is required for each ship to pass through the wormhole."`
+			`	Ivan then asks you to fill the rest of the scientists in on what you have found on the other side of the wormhole. After you finish talking, they begin to chatter among themselves, seeming to be most interested in the human beings which inhabit the region.`
+			`	They eventually come to the conclusion that you should return to this region of space and scan the ships of these humans so that they can learn more about them.`
+			choice
+				`	"Okay, I'm willing to do it."`
+					accept
+				`	"This seems too risky for me. You'll need to find someone else."`
+					decline
+					
+	on accept
+		event "clear arculus"
+	
+	npc "scan outfits"
+		government Remnant
+		personality staying uninterested
+		fleet
+			names remnant
+			variant
+				"Albatross"
+				"Starling"
+		dialog `You have finished scanning the Remnant ships in the system. You may now return to <destination> with the scanner logs.`
+	
+	on fail
+		event "reset arculus"
+	
+	on complete
+		event "reset arculus"
+		event "deep: scan log research" 21
+		dialog
+			`The team is visibly excited when you return with the scanner logs. "Give us a few weeks to look these over," Ivan says, "then we will contact you about what we want you to do next."`
```

Comment:
\`The team is visibly excited when you return with the scanner logs. "Give us a few weeks to look these over," Ivan says. "Then we will contact you about what we want you to do next."\`